README.md

```diff
@@ -1,34 +1,34 @@
-# TrebleShot open-source file and text sharing application
+# TrebleShot libre software file and text sharing app
 Send and receive files over available connections. [FAQ](https://github.com/genonbeta/TrebleShot/blob/master/FAQ.md).
 
 ## Get it on
 [<img src="https://f-droid.org/badge/get-it-on.png" width="230">](https://f-droid.org/packages/com.genonbeta.TrebleShot/) [<img src="assets/google-play-badge.png" width="230">](https://play.google.com/store/apps/details?id=com.genonbeta.TrebleShot)
 
 ## Main features
-* Pause and resume transfers even after something went wrong
+* Pause, resume and reconnect transfers
 * Share large files
-* Share multiple content from your video, photo, music and application libraries
-* No internet connection is required: setup a hotspot and connect to it using QR code
-* Share with multiple devices at the same time
+* Share all kinds of content, video, photo, music and app libraries
+* No Internet connection is required: Set up a hotspot and connect to it using a QR code
+* Share between multiple devices at the same time
 * Exchange texts of any kind and save them to TrebleShot
-* Share folders as they are for backup and other purposes
+* Share folders for backup and other purposes
 * Light UI: Works faster than its rivals on low-end devices
 * Speed-oriented: Minimal UI optimized for speed
-* Advanced features: network change handling, choosing network after speed measurement
+* Advanced features: Network change handling, choose network based on measured speed
 
 ## Project status
 [![Build Status](https://travis-ci.org/genonbeta/TrebleShot.svg)](https://travis-ci.org/genonbeta/TrebleShot)
 [![Translation Status](https://hosted.weblate.org/widgets/trebleshot/-/svg-badge.svg)](https://hosted.weblate.org/engage/TrebleShot/)
 
 ## Donate
-If you value this work, you can help it. There are only a few methods that work in Turkey,
-you can use Bitcoin or install Google Play version to do so (In App Billing method) (taxes will apply).
+If you value this work, you can help it along. Only a few methods of monetary contribution work in Turkey,
+you can use Bitcoin or install the Google Play version to do so (In App Billing method) (taxes will apply).
```

Comment:
What does " to do so (In App Billing method)" mean?

Reply:
It states that they can donate via Google Pay on their Android devices.